public/app/features/alerting/unified/components/rules/EditRuleGroupModal.tsx

```diff
@@ -64,6 +187,25 @@ export function EditCloudGroupModal(props: ModalProps): React.ReactElement {
     );
   };
 
+  const formAPI = useForm<FormValues>({
+    mode: 'onSubmit',
+    defaultValues,
+    shouldFocusError: true,
+  });
+  const {
+    handleSubmit,
+    register,
+    watch,
+    formState: { isDirty, errors },
+  } = formAPI;
+
+  const onInvalid = () => {
+    notifyApp.error('There are errors in the form. Please correct them and try again!');
```

Comment:
I think this is a bit unusual pattern for form validation in Grafana. Could we show an error below the table with rules?